ArduPlane/Attitude.cpp

```diff
@@ -572,6 +572,16 @@ bool Plane::suppress_throttle(void)
 
     if (control_mode==AUTO && 
         auto_state.takeoff_complete == false) {
+
+        if (is_flying() &&
+            millis() - started_flying_ms > 5000 && // been flying >5s in any mode
```

Comment:
won't this break bungee launches with parameters that lead to a greater than 5s throttle delay? We'll end up starting the throttle early and get tangled in the launch rope